backends/graphics/atari/atari_c2p-asm.S

```diff
@@ -0,0 +1,453 @@
+| C2P by Mikael Kalms (public domain)
+| See https://github.com/Kalmalyzer/kalms-c2p
+
+	.globl	_asm_c2p1x1_8
+	.globl	_asm_c2p1x1_8_rect
+
+
+	.text
+
+| void asm_c2p1x1_8(const char* pChunky, const char* pChunkyEnd, char* pScreen);
```

Comment:
Please add comments here too, explaining the purpose of these functions

Reply:
Done in the header file.